api/HTMLElement.json

```diff
@@ -1443,7 +1447,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "4"
```

Comment:
This is `isContentEditable`, a plain IDL attribute, so I trust the test results.